src/app/beer_garden/publish_request.py

```diff
@@ -35,18 +38,26 @@ def handle_event(event: Event):
             event.payload.namespace = None
             event.payload.system = None
             event.payload.system_version = None
-            event.paylaod.instance_name = None
+            event.payload.instance_name = None
             event.payload.command = None
             del event.payload.metadata["_publish"]
 
+        subscribers = []
+        for topic in get_all_topics():
+            if re.findall(topic.name, event.metadata["topic"]):
+                # get a list of subscribers for matching topic
+                subscribers = topic.subscribers
```

Comment:
You need to do a union of `subscribers` with `topic.subscribers`, because you could match on multiple topics. Especially since they can use regex in their topic.

Reply:
Fixed